test-run.py

```diff
@@ -100,7 +100,7 @@ def main_loop_parallel():
     lib.worker.print_greetings()
 
     color_stdout("\n", '=' * 86, "\n", schema='separator')
-    color_stdout("WORKR".ljust(6),     schema='t_name')
+    color_stdout("WORKER".ljust(7),    schema='t_name')
```

Comment:
It was written abbreviated intentionally to look as a table header for output below. But to be honest, I don’t remember before or after a worker id becomes three-digit long.